corehq/apps/app_manager/management/commands/set_linked_applications_uses_master_forms_true.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, print_function
+from __future__ import unicode_literals
+
+from django.core.management.base import BaseCommand
+from corehq.apps.app_manager.models import LinkedApplication, Application
+
+
+class Command(BaseCommand):
+    @staticmethod
+    def _iter_linked_apps():
```

Comment:
can you explain what this management command / the property you've added is intended to do? I'm a bit confused - it seems like it just sets this property for every app (that isn't a linked app) for all domains except vectorlink... But that means that only vectorlink's forms, or any forms from new linked apps will get into the view. Is that the intended effect? 
What are `master_form_ids`, and when might this get set in the rest of the code?

Reply:
So I really don't trust myself on friday afternoons, but I've fixed the logic. 

By `master_form_ids` I mean that it has the exact same `form_id` as the master app it is linked to. This exists because of an old bug that was never fully cleaned up. It was never cleaned up because changing a form id causes updates to crash the mobile app. If you have a better name, I'm open to suggestions.

What's supposed to be (and now is) happening here is that for every linked app that isn't one of vectorlink's, this is going to be set to true, which is going to take it out of this view. We know that vectorlink is safe, which makes this so the ticket you fixed before will still be resolved. As we figure out which ones are safe and which ones aren't we can flip the bool for more linked apps, and I believe that process is going to be resourced through the solutions team.

New linked apps are being created correctly, so they can always be false.

I added a comment to the property in the model.

Reply:
Maybe `uses_master_app_form_ids`? 
Can you also link to the original bug in the comment? The one you linked to isn't obviously related (and basically just links back to this PR).

Reply:
>`uses_master_app_form_ids`

That seems nice

>original bug

I have no idea where that ticket is and searching fogbugz got me nowhere. As far as I know, it lives in @calellowitz and @ctsims heads at this point. I think I'm going to leave this one for now since it has some context (as does this pr now) and let solutions update it when/if they make a ticket or trello card for it.